dockerfiles/haskell/3.0.0.0/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:22.04
+
+# Install necessary dependencies
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+    apt-utils curl build-essential libffi-dev libffi8 libgmp10 \
+    libgmp-dev libncurses-dev libncurses5 libtinfo5 pkg-config \
+    ca-certificates \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
Are any of these not required after install of GHCup, GHC, and Cabal?

Reply:
3.0.0.0 is from something deprecated, I'll change it

Reply:
Can curl, pkg-config, and ca-certificates be removed after install?

Reply:
oh after ghc you mean?

Reply:
Yeah

Reply:
I'll try it now :3